conseq/commands.py

```diff
@@ -365,12 +365,15 @@ def skip_remember(name):
         "Skipping export of %d executions which were filtered out via --exclude-remember",
         excluded,
     )
-    if dest_s3_path.startswith("s3://"):
-        log.info("Uploading artifact metadata to %s", dest_s3_path)
-        get_cas_remote().upload_str(dest_s3_path, out.getvalue())
+    if dest_gs_path.startswith("s3://"):
+        log.info("Uploading artifact metadata to %s", dest_gs_path)
+        get_cas_remote().upload_str(dest_gs_path, out.getvalue())
+    elif dest_gs_path.startswith("gs://"):
+        log.info("Uploading artifact metadata to %s", dest_gs_path)
+        get_cas_remote().upload_str(dest_gs_path, out.getvalue())
```

Comment:
These two code paths are identical correct? If they are, how about changing the test to an "or"?

It makes it clear that the same thing is going to happen in both cases. Otherwise as a reader I start looking very closely trying to find what's different between the handling of these files. (And the risk of one code path getting updated in the future but not the other one)